### DISCUSSION TITLE
Screen.safeArea & size.length property

### DISCUSSION THREAD

**Discussion by MalacTheLittle** [airsdk/Adobe-Runtime-Support#2415] [Q&A] [answered]
Hi,
what does `length` property represents in Screen.safeArea rectangle?
There are three  of 'em: size.length, bottomRight.length and topLeft.length.

**Comment by ajwfrost** [comment, marked as answer]
`Screen.safeArea` is a `Rectangle` object.. then
`Rectangle.size`, `Rectangle.bottomRight`, `Rectangle.topLeft` are all `Point` objects
These have a `length` property which is the length of a line drawn from (0,0) to the point location e.g. if `Screen.mainScreen.safeArea.bottomRight` was `(300, 400)` then `Screen.mainScreen.safeArea.bottomRight.length` would be 500 (square root of `300x300 + 400x400`).

https://help.adobe.com/en_US/FlashPlatform/reference/actionscript/3/flash/geom/Point.html#length

thanks
